pallets/staking-rewards/src/mock.rs

```diff
@@ -741,9 +743,9 @@ pub(crate) fn on_finalize(current_blk: BlockNumberFor<Test>) {
 pub fn default_test_ext() -> sp_io::TestExternalities {
     ExtBuilder::default()
         .initial_authorities(vec![
-            (VAL_1_STASH, VAL_1_CONTROLLER, VAL_1_AUTH_ID),
-            (VAL_2_STASH, VAL_2_CONTROLLER, VAL_2_AUTH_ID),
-            (VAL_3_STASH, VAL_3_CONTROLLER, VAL_3_AUTH_ID),
+            (VAL_1_STASH, VAL_1_STASH, VAL_1_AUTH_ID),
```

Comment:
Should we still accept 3 addresses there since controller deprecated?

Reply:
done